src/Form/Form.js

```diff
@@ -49,6 +67,7 @@ export default function Form() {
                 value={values.lastName}
                 name="lastName"
```

Comment:
always check the name in the input  and it should be the same in the useState. 
and always add value attribute here, so can see the text when is being changed.